src/ngraph/pass/graph_rewrite.cpp

```diff
@@ -138,30 +130,30 @@ static vector<regex> initialize_fusion_regexes()
     return regexes;
 }
 
-bool pass::GraphRewrite::is_enabled(const shared_ptr<pattern::Matcher>& m) const
+bool pass::GraphRewriteBase::is_enabled(const std::string& name) const
```

Comment:
@diyessi, please provide details (in the PR description or in the ticket) why we need all these changes here and below for GraphRewrite.

Reply:
The code was a mess, and using the pattern matcher as a way to look up a handler is based on op type is not a very good way to do things.